templates/product/product_list_base.html

```diff
@@ -98,6 +98,12 @@ <h3 class="d-md-none float-left filters-toggle">
   </div>
 {% endblock %}
 
+{% block topcontent %}
+  {% if object.background_image %}
+    <div class="row" id="product-list-image" style="background-image: url('{{ object.background_image.url }}')"></div>
```

Comment:
What do you think about making sure that the image is not too big (mainly dimensions)? (for example, if we have a staff member that would upload unoptimized pictures, like a picture taken from a mobile camera)

Currently `.container` has a `max-width` of `1140px`, maybe we could do something to prevent the page from being too data heavy?

Reply:
As an enhancement, background_image should be generated in various sizes, same as product images, so I'd not bother for now

Reply:
Okay then :+1: